components/builder-api/src/server/resources/pkgs.rs

```diff
@@ -1447,6 +1449,7 @@ fn has_circular_deps(req: &HttpRequest<AppState>,
     }
 }
 
+#[allow(clippy::redundant_closure)]
```

Comment:
This can be fixed. The issue with changing the code to `.map(BuilderPackageTarget::to_string)` is that `BuilderPackageTarget` doesn't itself implement `ToString`, [it derefs to PackageTarget](https://github.com/habitat-sh/builder/blob/master/components/builder-db/src/models/package.rs#L662-L666), which does. I'd suggest adding
```rust
impl ToString for BuilderPackageTarget {
    fn to_string(&self) -> String { self.to_string() }
}
```
to https://github.com/habitat-sh/builder/blob/master/components/builder-db/src/models/package.rs